src/map.ts

```diff
@@ -0,0 +1,117 @@
+import * as PIXI from "pixi.js";
+import map from "../public/tilemap.json";
+
+// Very simple state storage because who needs classes in javascript?
```

Comment:
:D Maybe the guyd who wants to check how much the movement shoudl be slowed by the sand tile he is on? :)

Reply:
😀